doc/source/composites.rst

```diff
@@ -2,48 +2,405 @@
     >>> img = to_image(composite)
     >>> img.invert([False, False, True])
     >>> img.stretch("linear")
     >>> img.gamma(1.7)
     >>> img.show()
 
+This part is called ``enhancement``, and is covered in more detail
+`enhancements <enhancements.html>`_ section.
+
+
+DifferenceCompositor
+--------------------
+
+:class:`DifferenceCompositor` calculates a difference of two datasets::
+
+    >>> from satpy.composites import DifferenceCompositor
+    >>> compositor = DifferenceCompositor("diffcomp")
+    >>> composite = compositor([local_scene[10.8], local_scene[12.0]])
+
+FillingCompositor
+-----------------
+
+:class:`FillingCompositor` fills the missing values in three datasets
+with the values of another dataset:::
+
+    >>> from satpy.composites import FillingCompositor
+    >>> compositor = FillingCompositor("fillcomp")
+    >>> filler = local_scene[0.6]
+    >>> data_with_holes_1 = local_scene['ch_a']
+    >>> data_with_holes_2 = local_scene['ch_b']
+    >>> data_with_holes_3 = local_scene['ch_c']
+    >>> composite = compositor([filler, data_with_holes_1, data_with_holes_2,
+    ...                         data_with_holes_3])
+
+PaletteCompositor
+------------------
+
+:class:`PaletteCompositor` creates a color version of a single channel
+categorical dataset using a colormap::
+
+    >>> from satpy.composites import PaletteCompositor
+    >>> compositor = PaletteCompositor("palcomp")
+    >>> composite = compositor([local_scene['cma'], local_scene['cma_pal']])
+
+The palette should have a single entry for all the (possible) values
+in the dataset mapping the value to an RGB triplet.  Typically the
+palette comes with the categorical (e.g. cloud mask) product that is
+being visualized.
+
+DayNightCompositor
+------------------
+
+:class:`DayNightCompositor` merges two different composites.  The
+first composite will be placed on the day-side of the scene, and the
+second one on the night side.  The transition from day to night is
+done by calculating solar zenith angle (SZA) weighed average of the
+two composites.  The SZA can optionally be given as third dataset, and
+if not given, the angles will be calculated.  Width of the blending
+zone can be defined when initializing the compositor (default values
+shown in the example below).
+
+    >>> from satpy.composites import DayNightCompositor
+    >>> compositor = DayNightCompositor("dnc", lim_low=85., lim_high=95.)
+    >>> composite = compositor([local_scene['true_color'],
+    ...                         local_scene['night_fog']])
+
+RealisticColors
+---------------
+
+:class:`RealisticColors` compositor is a special compositor that is
+used to create realistic near-true-color composite from MSG/SEVIRI
+data::
+
+    >>> from satpy.composites import RealisticColors
+    >>> compositor = RealisticColors("realcols", lim_low=85., lim_high=95.)
+    >>> composite = compositor([local_scene['VIS_006'],
+    ...                         local_scene['VIS_008'],
```

Comment:
This should be `VIS008`:

https://github.com/pytroll/satpy/blob/dff69ad5dc8373961111b4e1956f83e92a68618c/satpy/etc/readers/seviri_l1b_hrit.yaml#L287